lib/roo/excelx/sheet.rb

```diff
@@ -22,10 +22,6 @@ def cells
         @cells ||= @sheet.cells(@rels)
       end
 
-      def present_cells
```

Comment:
This is a public method, so maybe leave it in if people are using it? Or deprecate it? Or the next release will just need to be a breaking release, probably worth adding to the release notes if so.